lib/dialects/oracle/connection-manager.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const AbstractConnectionManager = require('../abstract/connection-manager');
+const Promise = require('../../promise');
+const sequelizeErrors = require('../../errors');
+const parserStore = require('../parserStore')('oracle');
+
+class ConnectionManager extends AbstractConnectionManager  {
+
+  constructor(dialect, sequelize) {
+    super(dialect, sequelize);
+
+    this.sequelize = sequelize;
+    this.sequelize.config.port = this.sequelize.config.port || 1521;
+    try {
+      if (sequelize.config.dialectModulePath) {
+        this.lib = require(sequelize.config.dialectModulePath);
+      } else {
+        this.lib = require('oracledb');
+        this.lib.maxRows = 1000;
+
+        if (sequelize.config && 'dialectOptions' in sequelize.config) {
+          const dialectOptions = sequelize.config.dialectOptions;
+          if (dialectOptions && 'maxRows' in dialectOptions) {
+            this.lib.maxRows = sequelize.config.dialectOptions.maxRows;
+          }
+
+          if (dialectOptions && 'fetchAsString' in dialectOptions) {
+            this.lib.fetchAsString = sequelize.config.dialectOptions.fetchAsString;
+          } else {
+            this.lib.fetchAsString = [2006];
```

Comment:
This [2006] looks like an arbitrary number. Can we specify a variable name instead or at least add a comment what this line means.

Reply:
It should be the constant`this.lib.CLOB`, if I'm not mistaken

Reply:
true, modified